### PR TITLE
Replaced Met CSV config with JSON config.

### LINKED ISSUE #27: Rewrite Met transform to use JSON received from Met's collections API

### PR DESCRIPTION
**Pull request by justinlittman**
closes #27 

Depends on https://github.com/sul-dlss/dlme-transform/pull/116

### CODE REVIEW

**Comment by justinlittman** [comment]
Done @anarchivist.

File: README.md [hunk 1]
```diff
@@ -119,3 +120,29 @@ to_field 'agg_has_view' do |_record, accumulator, context|
...
+}
+```
+
+##### Example Configuration
+
+```
+to_field('cho_title'), extract_json('.title', )
+to_field 'agg_is_shown_at' do |_record, accumulator, context|
+  accumulator << transform_values(context,
+                                  'wr_id' => [extract_json('.thumbnail.resource')
```
Comment (jcoyne) [inline README.md]: missing a `]` here I think.

File: met_config.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+end
+
+to_field 'agg_provider', provider
+
+# MET Museum
+to_field 'id', extract_json('.objectID'), lambda { |_record, accumulator, context|
+  accumulator.map! { |bare_id| identifier_with_prefix(context, bare_id.to_s) }
+}
+
+# Note: Met JSON uses blanks ("") instead of nulls.
```
Comment (jcoyne) [inline met_config.rb]: This is really helpful context 👍

File: README.md (hunk 1 above)
Comment (justinlittman) [reply README.md]: Hah, I copied it from the XML example above.

**Pull request merged**